src/mutator_test.cc

```diff
@@ -577,6 +578,75 @@ TYPED_TEST(MutatorTypedTest, FailedMutations) {
   EXPECT_LT(crossovers, 100u);
 }
 
+TYPED_TEST(MutatorTypedTest, FieldMutator) {
+  constexpr int kNoMutation = 1;
+  constexpr int kRegularMutation = 2;
+  constexpr int kCustomMutation = 4;
+  constexpr int kAllCases = kNoMutation | kRegularMutation | kCustomMutation;
+
+  constexpr char kIndicatorString[] = "0123456789abcdef";
+
+  TestMutator mutator(false);
+  typename TestFixture::Message message;
+  TestMutator::RegisterCustomMutation(
+      message.GetDescriptor()->FindFieldByName("optional_string"),
+      [kIndicatorString](protobuf::Message* message){
+        typename TestFixture::Message* test_message =
+            dynamic_cast<typename TestFixture::Message*>(message);
+        test_message->set_optional_string(kIndicatorString);
+      });
+  TestMutator::RegisterCustomMutation(
+      message.GetDescriptor()->FindFieldByName("optional_bytes"),
```

Comment:
optional_bytes looks just redundant and increase the test almost twice.
Could you please remove it?

Reply:
Done. Note that the time required for the test to pass is substantially longer (100 seconds instead of 18).